### PR TITLE
Cosmetic fixes and documentation

### PR DESCRIPTION
**Pull request by Shtoyan**
Ola! Just a few touch-ups to make the repo more accessible.

- Added default config file.
- Added proper LICENSE.
- Added README with minimal required information.
- Added links to repo in source files and config.

### CODE REVIEW

File: Configs/MutLoadedLevel.ini [hunk 1]
```diff
@@ -0,0 +1,52 @@
+;= Home Repo: https://github.com/EliotVU/UT2004-MutLoadedLevel
```
Comment (EliotVU) [inline Configs/MutLoadedLevel.ini:1]: What does the character = after a ; do?

Comment (Shtoyan) [reply Configs/MutLoadedLevel.ini:1]: That's a handy hack, so comments never delete on config overwrite.

Comment (EliotVU) [reply Configs/MutLoadedLevel.ini:1]: Ah nice, thanks!

**Comment by EliotVU** [comment]
Seems good to go!, thank you very much!

**Comment by Shtoyan** [comment]
> Seems good to go!, thank you very much!

No you.

**Pull request merged**